app/assets/stylesheets/external-links.scss

```diff
@@ -0,0 +1,20 @@
+
+#external-links {
+
+  .media {
+    .icon {
+      fill: #4a4a4a;
+    }
+  }
+
+  li {
+    margin-bottom: 0.5rem;
```

Comment:
I was not sure if it was better to add `mb-2` to each element of the list or to add a css rule. For me css sounded as a more logical solution. You only need to change one line to affect all of the list elements while in case of helpers you have a possibility of forgetting and introducing bugs.

Reply:
@Macxim what's your take on this - single change of css or adding helpers to all menus?

Reply:
I've changed from css changes to adding helpers

Reply:
I think, with the rise of utility-based CSS frameworks, the idea is to write the least amount of pure CSS as possible. Basically the restrictions are there for reason: to prevent users from doing: `font-size: 12px`, next to a smaller text of: `font-size: 9px`, next to a sightly bigger text of `font-size: 10px`, but then that one button should be something in between, so let's add `font-size: 11px` only here.

It's also easier to open `.html.erb` file and see this:

```html
<a class="button is-primary is-outlined m-2 px-4 py-2">
  Subscribe
</a>
```

Than to see this:
```html
<a class="subscribe-button button is-primary is-outlined m-2 py-2">
  Subscribe
</a>
```

```css
.subscribe-button {
  padding-left: 10px;
  padding-right: 10px;
}
```

That's at least how I see it. Tailwind CSS is a leader in this "school of thought". :)